src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -292,6 +300,84 @@ message MethodConfigProto {
 
 }
 
+// `CodeSampleValues` defines sets of parameter values to be used in
+// generating samples.
+message CodeSampleValues {
+  // An identifier, unique to this method, for this set of values.
+  string id = 1;
+
+  // A short, user-visible name for this set of values. This name may
+  // be used in a UI menu, for example.
+  string title = 2;
+
+  // A longer description for this set of values. This name may be
+  // used in a UI hoverbox, for example.
+  string description = 3;
+
+  // A set of parameter values to be used together in samples. Each
+  // member of `parameters` can be specified in one of two formats:
```

Comment:
Note: our current initialization fields have the following grammar (copied from an internal doc), which has special syntax for maps and arrays, and which doesn't support the special syntax for `FIELD` that you propose:

```
<field-path> ::= <field> | <field>"." <field-path>
<field> ::= <field-name> | <field-name> <array-index> | <field-name> <map-key>
<array-index> ::= "[" <unsigned integer> "]"
<map-key> ::= "{" <map-key-name> "}"
<map-key-name> ::= [a-zA-Z0-9_]*
```

Reply:
I wasn't proposing it. I was trying to capture the syntax for formatted field names as it already exists in the [code](https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml#L85), but I used the wrong field placeholder! Ooops.

Fixed.